tests/translator/output/translate_convert_metadata.json

```diff
@@ -0,0 +1,93 @@
+{
+  "Resources": {
+    "Images": {
+      "Type": "AWS::S3::Bucket",
```

Comment:
The bucket is defined in the template above, I don't think we want SAM to mutate that resource. I know we do today, but that is in very specific use-cases (adding events triggers).

Reply:
I'm not 100% seeing how this resource is being mutated, since it exists previously. What am I overlooking?

Reply:
Note, to save back and forth:

If we want to exclude certain resource types from the metadata passthrough, that's fine. I'll need a list of resource types that would apply here. I'm a little bit in the dark.